script/bytecode-deploy/build-output/ColdStorageAddressBookPlugin.json

```diff
@@ -0,0 +1 @@
d2d1857b21182ad8c439f","urls":["bzz-raw://812c9a182acfff9d1a942ac6450af4c8f60b8b4bc50b7fccdef4baa109b7f047","dweb:/ipfs/QmczW9rNqdViG3YkJP7fKCTbwzMHS6oe5mH3f92UQjSqki"],"license":"GPL-3.0-or-later"},"src/libs/RecipientAddressLib.sol":{"keccak256":"0x021ef5f99b5d7a312bed13b9c7b9785de5e7744baba6d5041ef6d6ebee1b1e8e","urls":["bzz-raw://730a4ddb81c9a60c3dc4170af85e5886942b8a0fc8d3ae5bf8c6a9452ca88cc2","dweb:/ipfs/QmY2CtzDhtK3CfEH71fydMo3REh5Cp6r14Bdhvij6LDGAt"],"license":"GPL-3.0-or-later"},"src/msca/6900/shared/common/Errors.sol":{"keccak256":"0x6c0697fb3959257772b7fa5f277f2a42b289201f1db68920eb4a71e45e7b06f5","urls":["bzz-raw://7ff2590301d8fb136a1a494081627e53bab34187abe8b5ba304a9bf67878c70d","dweb:/ipfs/QmY6kg1DuQNEZHMGLRoHLzykJfs3oioZTm91CBZefKgeg8"],"license":"GPL-3.0-or-later"},"src/msca/6900/v0.7/common/PluginManifest.sol":{"keccak256":"0x353a9c989f7c192f72f43628ad39ec45a7c7d73df8d76ebb83881b8278e5f8da","urls":["bzz-raw://944bf9f631abe512633a5d0647e521ce69ef91faba3173a64cc453672d1afbe1","dweb:/ipfs/QmfFycsZdU6mVpXP9sr7H9CzU2CbE2iZb13BQxa5gh8N3n"],"license":"GPL-3.0-or-later"},"src/msca/6900/v0.7/common/Structs.sol":{"keccak256":"0xf02819076c22bce75decaa5250607d9ac2c369e5ff978b5f9db437f23e84de46","urls":["bzz-raw://46d85390b7f994c4240c6ca2bc7e7fd68a6085612b9bcab707a722195093ba59","dweb:/ipfs/QmYYn3FywFNb3g7THmUfGKFo2K1vr5FE3bWNeGXCzeb1Dj"],"license":"GPL-3.0-or-later"},"src/msca/6900/v0.7/interfaces/IPlugin.sol":{"keccak256":"0x7ed24c53e66bd4b93e7eea2b2673bdb1799e5c90ff0749d4af61e898cf548be1","urls":["bzz-raw://cbe3e1795a01016bc5c92f77158b2e00b8035c3d763ecd21615e2b653da6dced","dweb:/ipfs/QmUY5DsFHpPh4onXbZX6cNdptYeXPDD3rSye9aqqKH5WDS"],"license":"GPL-3.0-or-later"},"src/msca/6900/v0.7/interfaces/IStandardExecutor.sol":{"keccak256":"0xcad6f9743a88912406cfc9923d07f145aa0015745410161db86ec363d4e2a978","urls":["bzz-raw://084e70e5ffb906b83e83dfc456ba656adc2f2de683d9b07a8a537e78163c5f0c","dweb:/ipfs/QmTj8TCCsg8JrJJGgWMjZrWHE1137a1fciMo7yHh5E76xp"],"license":"GPL-3.0-or-later"},"src/msca/6900/v0.7/plugins/BasePlugin.sol":{"keccak256":"0x52f0cd9282a1973905813a233100452d5708aecf30b693ce3e34ce43f1cb072a","urls":["bzz-raw://69e5fcd52522ec830e55ec120a7f5c82dc68aab1bf765b6054d79b449feb6d55","dweb:/ipfs/QmPBQRfw9sNtpcFKnYqBZgxPSNUQNjQZLvgV8W6AiPQCfk"],"license":"GPL-3.0-or-later"},"src/msca/6900/v0.7/plugins/v1_0_0/addressbook/ColdStorageAddressBookPlugin.sol":{"keccak256":"0xb70d286929ce5e0c89e5e50163d8e9f52b4bc9436e465fd28d46d4dac3eb766f","urls":["bzz-raw://aaf8fd8c498257ff87515fc8803602e8daeae998f88e33ced297ddf69a9ee224","dweb:/ipfs/QmX5oe5C7EfAsEev3KDyeJxQNPVVbJtBcKR8UvQJ8RzHqv"],"license":"GPL-3.0-or-later"},"src/msca/6900/v0.7/plugins/v1_0_0/addressbook/IAddressBookPlugin.sol":{"keccak256":"0xe99651bc115182e10ac393c1bb88b59a0a3094ace6554f8691689331eaac5645","urls":["bzz-raw://b4c5809ddcb307d7d03781b6271e5942a37fb6cb085c31e4b38eb7ba16ebd9a0","dweb:/ipfs/QmchqWnby2S5PaqzURzAUYGYuHREZzREkUrW2DTViE4EA3"],"license":"GPL-3.0-or-later"}},"version":1},"id":18}
```

Comment:
maybe also add a readme [here](https://github.com/circlefin/buidl-wallet-contracts/tree/master/script) how we generate this file so others can follow